llvm/test/tools/llvm-dlltool/arm64ec.test

```diff
@@ -32,7 +32,10 @@ ARMAP2-NEXT: __imp_aux_func in test.dll
 ARMAP2-NEXT: __imp_func in test.dll
 ARMAP2-NEXT: func in test.dll
 
-RUN: not llvm-dlltool -m arm64 -d test.def -n test2.def -l test2.lib 2>&1 | FileCheck --check-prefix=ERR %s
+RUN: llvm-dlltool -m arm64ec -d test.def -N test2.def -l test3.lib
```

Comment:
What does this line test, isn't this the exact same test as what's above on line 19?

Reply:
It was meant to test the long form, but I failed to commit the final version. I pushed a fixup.